src/en/news/blog/2022/centralized_logging/index.md

```diff
@@ -0,0 +1,114 @@
+---
+title: "Centralized Logging on Ceph Dashboard"
+date: "2022-11-23"
+author: "Nizamudeen A"
+image: "images/banner.jpg"
+tags:
+  - "ceph"
+  - "dashboard"
+  - "logs"
+  - "monitoring"
+  - "loki"
+  - "promtail"
+  - "grafana"
+  - "centralized logging"
+---
+
+_Disclaimer: This article proclaims Loki as a hero._
+
+## What are we trying to solve?
+
+Ceph, as a distributed system, usually generates a lot of logs. And these logs can be from mons, mgr, osds or any of the clients of ceph.
```

Comment:
add a comma after osds.

Reply:
`OSDs` since it's an acronym, and capitalize Ceph.